src/Router/DefaultRouter.php

```diff
@@ -11,6 +15,6 @@ class DefaultRouter extends BaseRouter
      */
     public function getRoute(Request $request)
     {
-        return new Route('DefaultController', 'handle');
+        return new Route('Joindin\Api\Controller\DefaultController', 'handle');
```

Comment:
Could this be `return new Route(DefaultController::class, 'handle');` instead and import the path as a `use` statement instead?

Reply:
That would be better, will do in a followup PR